exopy_hqc_legacy/tasks/tasks/instr/tune_IQ_mixer.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2018 by ExopyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Tasks to set the parameters of arbitrary waveform generators.
+
+"""
+import logging
+import numbers
+import numpy as np
+
+from atom.api import (Enum, Unicode, Value)
+
+from exopy.tasks.api import (InstrumentTask, validators)
+import exopy_hqc_legacy.instruments.drivers.dll.SA124B as sa
+
+SA_SWEEPING = 0x0
+SA_REAL_TIME = 0x1
+
+
+class TuneIQMixerTask(InstrumentTask):
+    """ Task to tune an IQ mixer in SSB
+        Implicit use of a SignalHound spectrum analyzer
+        Tunes channels I and Q DC offset, relative delay and voltage
+        to suppress LO leakage and unwanted sideband
+        TODO: handle task with two instruments: AWG AND Spectrum analyzer
+        TODO: implement realtime sweep for better SNR
+
+    """
+
+    # Get user inputs
```

Comment:
FYI Sphinx will only integrate comments starting with `#:` in the docs, so out of habit I use those for atom members